driver_cpl/cime_config/config_component.xml

```diff
@@ -409,7 +409,7 @@
   <entry id="DOUT_S"> 
     <type>logical</type>
     <valid_values>TRUE,FALSE</valid_values>
-    <default_value>TRUE</default_value>
+    <default_value>FALSE</default_value>
```

Comment:
I think that we need a way to differentiate between models in the
config_component.xml file - perhaps an additional field
<model></model>    would do the trick.   This in the case where we want
different defaults we can add a model attribute to the
<default_value> field.      This would resolve the input arguments but we
still need to figure out how to handle the differences in shr_pio_mod.

On Thu, Jan 14, 2016 at 5:06 PM, Andy Salinger <notifications@github.com>
wrote:

> In driver_cpl/cime_config/config_component.xml
> <https://github.com/ESMCI/cime/pull/33#discussion_r49805378>:
>
> > @@ -409,7 +409,7 @@
> >    <entry id="DOUT_S">
> >      <type>logical</type>
> >      <valid_values>TRUE,FALSE</valid_values>
> > -    <default_value>TRUE</default_value>
> > +    <default_value>FALSE</default_value>
>
> Currently, we prefer it FALSE. Mark turned DOUT_S to false a few months
> ago, and seems to prefer it. His reasoning is because he prefers the
> organization of the log files without this step. I like it because I can
> run case.run, instead of case.submit (I hope I have it correct). No one has
> complained that I know of.
>
> In the medium term, we will probably need to have a way of different
> defaults, since this is likely not the last case.
>
> In the short term, you can pick one, and one team has to deal temporarily,
> but the next coin-flip goes the other way.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ESMCI/cime/pull/33/files#r49805378>.
>



-- 
Jim Edwards

CESM Software Engineer
National Center for Atmospheric Research
Boulder, CO